src/diffusers/utils/remote_utils.py

```diff
@@ -0,0 +1,323 @@
+# coding=utf-8
+# Copyright 2025 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: `imghdr` is deprecated in Python 3.13 🙄
```

Comment:
Do they have a way to determine that otherwise in Python3.13 then?

Reply:
https://docs.python.org/3/library/imghdr.html
We'll have to use a third party module